extensions/docker/src/container_detail.tsx

```diff
@@ -21,21 +22,26 @@ export default function ContainerDetail({ docker, containerId }: { docker: Docke
               <Detail.Metadata.TagList.Item
                 text={containerInfo.State.Status}
                 color={containerInfo.State.Running ? Color.Green : Color.Yellow}
+                key={randomUUID()}
               />
             </Detail.Metadata.TagList>
             <Detail.Metadata.TagList title="Command">
-              <Detail.Metadata.TagList.Item text={containerInfo.Config.Cmd?.join(' ')} />
+              <Detail.Metadata.TagList.Item text={containerInfo.Config.Cmd?.join(' ')} key={randomUUID()} />
             </Detail.Metadata.TagList>
             {containerInfo.Config.ExposedPorts && (
               <Detail.Metadata.TagList title="Ports">
                 {Object.keys(containerInfo.Config.ExposedPorts).map((p) => (
-                  <Detail.Metadata.TagList.Item text={p} color={Color.PrimaryText} />
+                  <Detail.Metadata.TagList.Item text={p} color={Color.PrimaryText} key={randomUUID()} />
```

Comment:
As per @gebeto comment below, this could be written as:

```typescript
{Object.keys(containerInfo.Config.ExposedPorts).map((p, idx) => (
  <Detail.Metadata.TagList.Item text={p} color={Color.PrimaryText} key={idx} />
))}
```

Reply:
you are right! fixed